torchgeo/datasets/geo.py

```diff
@@ -350,19 +350,18 @@ def __init__(
             cache: if True, cache file handle to speed up repeated sampling
 
         Raises:
-            FileNotFoundError: if no files are found in ``root``
+            FileNotFoundError: if no files are found in ``paths``
```

Comment:
Need to add something like:
```rst
.. versionchanged:: 0.5
   *root* was renamed to *paths*
```
Same for other datasets

Reply:
I wish. I don't think that's possible.

Reply:
Should be fixed